calico_versioned_docs/version-3.30/variables.js

```diff
@@ -20,7 +20,7 @@ const variables = {
   manifestsUrl: 'https://raw.githubusercontent.com/projectcalico/calico/v3.30.3',
   releases,
   registry: '',
-  vppbranch: 'v3.29.0',
+  vppbranch: 'v3.30.0',
```

Comment:
@lwr20 Do we need additional testing here? I recall in the past that we just relied on @onong to test.